### ISSUE TITLE
Menu Issue with 7" Screen.

### ISSUE THREAD

**Issue by K7JLT** [km4ack/pi-build#94]
The 1st Build-a-Pi menu doesn't allow selection of options because the window is off the bottom of the screen.

**Comment by km4ack** [comment]
Are you able to set a higher screen resolution during the build so you can see the bottom?

**Comment by K7JLT** [comment]
No, the display (original R Pi display) is at highest resolution.

**Comment by kg4cfx** [comment]
I am having the same issue. What do i do to fix it.

**Comment by chrisbedwards56** [comment]
I’m having the same issue on hotspot tools menu.  Also conky resets off of the screen then doesn’t reappear after running the resize conky.
